interface-elements-for-desktop/articles/rich-text-editor/track-changes/enable-track-changes.md

```diff
@@ -0,0 +1,59 @@
+# Enable Track Changes
+
+The Rich Text Editor allows you to use Track Changes to make revisions in a document.
+
+## Turn On Track Changes
+
+On the **Review** [tab](text-editor-ui/ribbon-interface.md), in the **Tracking** group, click the **Track Changes** button.
```

Comment:
Same here - Why do you write it in this order?

Reply:
I follow the Microsoft Manual of Style

> _Microsoft style_
> Under Table Tools, on the Layout tab, in the Data group, click Repeat Header Rows.
> _Not Microsoft style_
> Click the Layout tab under Table Tools, and then in the Data group, click Repeat Header Rows.

Reply:
Ok